src/app/containers/ChartbeatAnalytics/index.test.jsx

```diff
@@ -120,35 +118,31 @@ describe('Charbeats Analytics Container', () => {
       {},
     );
     expect(testUtils.getConfig).toHaveBeenCalledTimes(1);
-    expect(tree).toMatchSnapshot();
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
can we not use the shouldMatchSnapshot helper from pasammead-test-helpers?

Reply:
There are other asserts in the test, so I'd rather make that change in another PR if possible